projects/plugins/jetpack/extensions/blocks/cookie-consent/index.js

```diff
@@ -0,0 +1,65 @@
+import { ExternalLink } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { getIconColor } from '../../shared/block-icons';
+import attributes from './attributes';
+import edit from './edit';
+import icon from './icon';
+import save from './save';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'cookie-consent';
+export const title = __( 'Cookie Consent', 'jetpack' );
+export const cookieName = 'eucookielaw';
+export const settings = {
+	title,
+	description: (
```

Comment:
This will add another instance of #26792 that would need fixing soon.

Reply:
Good call. Given our timeline (WordPress 6.2 landing in March), that's probably something we want to fix in here before we merge this.